src/Quiz/Quiz.js

```diff
@@ -12,7 +13,6 @@ class Quiz extends React.Component {
   }
 
   componentDidMount(){
-
     if(this.inputRef.current && this.props.active){
       this.inputRef.current.focus();
     }
```

Comment:
it could be one function that are called by componentDidMount and componentDidUpdate